.github/workflows/test.yml

```diff
@@ -210,10 +209,9 @@ jobs:
           working-directory: cli
           args: --new-from-rev=${{ github.event.pull_request.base.sha || 'HEAD~1' }}
 
-  go_unit:
-    name: Go Unit Tests
+  turborepo_tests:
+    name: Go Unit, Integration, and E2E Tests
     needs: determine_jobs
-    if: needs.determine_jobs.outputs.go == 'true'
```

Comment:
I think we still want to restrict this to run only when either turborepo (Rust or Go) changes

Reply:
i was thinking turborepo can just do this. "always run, but get cache hit" when source files haven't changed

Reply:
My only concern with that is that we'll be waiting on a turbo build before we can decide if we should run.